plugin/pkg/admission/limitranger/admission.go

```diff
@@ -54,12 +54,21 @@ type limitRanger struct {
 
 // Admit admits resources into cluster that do not violate any defined LimitRange in the namespace
 func (l *limitRanger) Admit(a admission.Attributes) (err error) {
-
 	// Ignore all calls to subresources
 	if a.GetSubresource() != "" {
 		return nil
 	}
 
+	// NOTE:
```

Comment:
In addition to this comment, why not just go ahead and check that the pod's resources haven't changed? That will guarantee that this will not break in the way you're worried about.